app/controllers/incomeSources/manage/ManageIncomeSourceDetailsController.scala

```diff
@@ -117,86 +154,148 @@ class ManageIncomeSourceDetailsController @Inject()(val view: ManageSelfEmployme
     }
   }
 
+  def convertCashOrAccrualsFlagIntoString(cashOrAccrualsFlag: Option[Boolean]): Option[String] = {
```

Comment:
this is model specific conversion and preferable to keep it within actual model, ie you can move this into relevant companion object